//
// Created by sunbo on 2020/9/28.
//

#ifndef _KIESHOP_TOOL_CELLAR_HANDLER_H
#define _KIESHOP_TOOL_CELLAR_HANDLER_H

#include "../gen-cpp/tool_cellar_types.h"

KieShop::tool_cellar::RequestLoggingResponse logRequest(const KieShop::tool_cellar::RequestLoggingRequest& request);



#endif //_KIESHOP_TOOL_CELLAR_HANDLER_H
